src/UhdmAst.cpp

```diff
@@ -441,23 +441,32 @@ AstNodeDType* getDType(vpiHandle obj_h, UhdmShared& shared) {
         // todo: static/dynamic/assoc/queue
         auto rand_type = vpi_get(vpiRandType, obj_h);
         // todo: rand/randc/notrand
-        AstNodeDType* element_dtype = nullptr;
+        AstNodeDType* element_dtypep = nullptr;
 
         vpiHandle itr = vpi_iterate(vpiReg, obj_h);
         while (vpiHandle member_h = vpi_scan(itr)) {
-            std::string type_name;
-            auto type_h = vpi_handle(vpiTypespec, member_h);
```

Comment:
Please bring back this typespec handling, it seems that removing it broke Earlgrey and Ibex simulations (see Actions). If there is a typespec, use it, if there isn't one, do the rest of your code.

(At least I hope that's it, it could be a harder-to-find issue)

Reply:
I am investigating fails in these simulations now. I will check it.

Reply:
You were right. I corrected it. Thanks.